src/Models/Views/MovieViewModel.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using TrainingNet.Models.DataBase;
+
+namespace TrainingNet.Models.Views
+{
+    public class MovieViewModel
+    {
+        public MovieViewModel(){
+            
```

Comment:
Since the constructor is empty, it might be simpler to use a single line
`public MovieViewModel() { } `
(or you can just delete it. In this case it's not doing anything so it might be better to just remove it. There are some cases in which an empty constructor could be declared, but i think it's not needed here)